test/colorado_lottery_test.rb

```diff
@@ -0,0 +1,72 @@
+  def setup
+    @lottery = ColoradoLottery.new
+    @pick_4 = Game.new('Pick 4', 2)
+    @mega_millions = Game.new('Mega Millions', 5, true)
+    @cash_5 = Game.new('Cash 5', 1)
+    @alexander = Contestant.new({first_name: 'Alexander', last_name: 'Aigades', age: 28, state_of_residence: 'CO', spending_money: 10})
+    @benjamin = Contestant.new({first_name: 'Benjamin', last_name: 'Franklin', age: 17, state_of_residence: 'PA', spending_money: 100})
+    @frederick = Contestant.new({first_name:  'Frederick', last_name: 'Douglas', age: 55, state_of_residence: 'NY', spending_money: 20})
+    @winston = Contestant.new({first_name: 'Winston', last_name: 'Churchill', age: 18, state_of_residence: 'CO', spending_money: 5})
+    @grace = Contestant.new({first_name: 'Grace', last_name: 'Hopper', age: 20, state_of_residence: 'CO',spending_money: 20})
+  end
+
+  def test_it_exists
+    assert_instance_of ColoradoLottery, @lottery
+  end
+
+  def test_it_has_attributes
+    assert_equal ({}), @lottery.registered_contestants
+    assert_equal [], @lottery.winners
+    assert_equal ({}), @lottery.current_contestants
+  end
+
+  def test_it_can_be_interest_and_of_age
+    @alexander.add_game_interest('Pick 4')
+    @alexander.add_game_interest('Mega Millions')
+    @benjamin.add_game_interest('Mega Millions')
+    assert_equal true,  @lottery.interested_and_18?(@alexander, @pick_4)
+    assert_equal false, @lottery.interested_and_18?(@benjamin, @mega_millions)
+    assert_equal false, @lottery.interested_and_18?(@alexander, @cash_5)
+  end
+
+  def test_it_can_register
+    @alexander.add_game_interest('Pick 4')
+    @alexander.add_game_interest('Mega Millions')
+    @frederick.add_game_interest('Mega Millions')
+    @winston.add_game_interest('Cash 5')
+    @winston.add_game_interest('Mega Millions')
+    @benjamin.add_game_interest('Mega Millions')
+    assert_equal true, @lottery.can_register?(@alexander, @pick_4)
+    assert_equal true, @lottery.can_register?(@frederick, @mega_millions)
+    assert_equal false, @lottery.can_register?(@benjamin, @mega_millions)
+    assert_equal false, @lottery.can_register?(@frederick, @cash_5)
+  end
+
+  def test_it_has_registered_contestants
+    @alexander.add_game_interest('Pick 4')
+    @alexander.add_game_interest('Mega Millions')
+    @lottery.register_contestant(@alexander, @pick_4)
+    @lottery.register_contestant(@alexander, @mega_millions)
+
+    assert_equal ({"Pick_4" => @alexander, "Mega Millions" => @alexander}), @lottery.registered_contestants
+
+    @grace.add_game_interest('Mega Millions')
+    @grace.add_game_interest('Cash 5')
+    @grace.add_game_interest('Pick 4')
+    @lottery.register_contestant(@grace, @mega_millions)
+    @lottery.register_contestant(@grace, @cash_5)
+    @lottery.register_contestant(@grace, @pick_4)
+    expected = {"Pick 4" => @alexander, @grace, "Mega Millions" => @alexander, @grace, "Cash 5" => @grace}
+
+    assert_equal expected, @lottery.registered_contestants
```

Comment:
i'm not sure you ran this test. it has a couple of syntax errors (you should have an array around alexander and grace for pick 4 and mega millions), so it won't even run. 

You also deviated from the interaction pattern here, which would have been okay if it enforced exactly the same behavior, but since you're asserting a different data type as the value, it doesn't enforce the same behavior. for these independent challenges, I really recommend just sticking with copying the interaction pattern and using the exact same setup so you know you're getting the same behavior.